### PR TITLE
Improve quote handling to make language .ini files valid

### PR DESCRIPTION
**Pull request by demiankatz**
This PR implements a change suggested in #3048 to improve Lokalise compatibility. I think this change is probably worth making whether or not it impacts Lokalise, though, since it gets our .ini files closer to validity in terms of generic .ini standards. I also caught and fixed quite a few quote-related problems (including removing some comments that are not real translations) in various files as a result of running this new normalization.

TODO
- [x] Run full test suite before merging

### CODE REVIEW

**Comment by demiankatz** [comment]
@EreMaijala, as discussed on Slack, I've added escaping of quotes inside the same type of quote, to make this create fully standards-compliant .ini files. I don't have time to run the full test suite today, but if this looks good to you, I can do so tomorrow.

**Comment by demiankatz** [comment]
Full test suite is passing.

File: languages/ca.ini [hunk 1]
```diff
@@ -1372,7 +1372,7 @@ username_only_letters_numbers_and_basic_punctuation = "Lletres, números i puntu
 username_only_numeric = "Només números"
 verification_done = "La teva adreça de correu electrònic s'ha verificat correctament."
 verification_email_change_sent = "S'han enviat instruccions de verificació de l'adreça de correu electrònic a la nova adreça de correu electrònic. Heu de verificar l'adreça abans que el canvi tingui efecte."
-verification_email_notification = ""S'ha fet una sol·licitud per verificar la vostra adreça de correu electrònic del vostre compte amb %%library%%."
+verification_email_notification = "S'ha fet una sol·licitud per verificar la vostra adreça de correu electrònic del vostre compte amb %%library%%."
```
Comment (EreMaijala) [inline languages/ca.ini:1375]: Shouldn't this one be changed to single quotes as well?

**Review by EreMaijala — CHANGES_REQUESTED** [review]

File: languages/ca.ini (hunk 1 above)
Comment (EreMaijala) [reply languages/ca.ini:1375]: No, I was mistaken, all good!

**Review by EreMaijala — APPROVED** [review]

**Pull request merged**